render.go

```diff
@@ -49,16 +49,14 @@ const (
 	endState
 )
 
-// validFlags holds the list of valid flags, for quick checkup.
-// Valid flags are 'bdoxXeEfFgGrts%'
-var validFlags = map[byte]struct{}{
-	'b': struct{}{}, 'd': struct{}{}, 'o': struct{}{}, 'x': struct{}{}, 'X': struct{}{},
-	'e': struct{}{}, 'E': struct{}{}, 'f': struct{}{}, 'F': struct{}{}, 'g': struct{}{},
-	'G': struct{}{}, 'r': struct{}{}, 't': struct{}{}, 's': struct{}{}, '%': struct{}{}}
+// validFlags are 'bdoxXeEfFgGrts%'
+func validFlag(b byte) bool {
+	return (b == 'b' || b == 'd' || b == 'o' || b == 'x' || b == 'X' || b == 'e' || b == 'E' || b == 'f' || b == 'F' || b == 'g' || b == 'G' || b == 'r' || b == 't' || b == 's' || b == '%')
+}
 
-var isDigit = map[byte]struct{}{
-	'0': struct{}{}, '1': struct{}{}, '2': struct{}{}, '3': struct{}{}, '4': struct{}{},
-	'5': struct{}{}, '6': struct{}{}, '7': struct{}{}, '8': struct{}{}, '9': struct{}{},
+func isDigit(d byte) bool {
+	return (d == '0' || d == '1' || d == '2' || d == '3' || d == '4' || d == '5' ||
```

Comment:
Would 		
```
return '0' <=d && d <= '9'
 ```

Be better?

Reply:
Didn't see your comment before merging. Let me benchmark, but probably.

Reply:
It is. Included that change in pull request #9 

Benchmark results (digit is original, digit2 is your suggestion):

BenchmarkDigit-8                        2000000000               1.82 ns/op            0 B/op          0 allocs/op
BenchmarkDigit2-8                       2000000000               1.23 ns/op            0 B/op          0 allocs/op